testsuite/smoke-test/src/state_sync_v2.rs

```diff
@@ -384,16 +379,13 @@ async fn test_all_validator_failures(mut swarm: LocalSwarm) {
 #[ignore]
 async fn test_single_validator_failure() {
     // Create a swarm of 1 validator
-    let mut swarm = new_local_swarm_with_aptos(1).await;
-    swarm.launch().await.unwrap();
-
-    // Enable state sync v2 and reboot the node
-    let validator = swarm.validators_mut().next().unwrap();
-    let mut config = validator.config().clone();
-    config.state_sync.state_sync_driver.enable_state_sync_v2 = true;
-    config.save(validator.config_path()).unwrap();
-    validator.stop();
-    swarm.launch().await.unwrap();
```

Comment:
for example, these lines were starting a node 3 times. (in new_local_swarm_with_aptos, swarm.launch() and swarm.launch after reconfig)
now they are starting it only once.